api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -193,6 +193,11 @@ message HttpConnectionManager {
     //   Such a constraint is inherent to OpenCensus itself. It cannot be overcome without changes
     //   on OpenCensus side.
     config.trace.v3.Tracing.Http provider = 9;
+
+    // Trace reason is decided by request_id by default. If true, these sampling strategy decisions
+    // will be decided by only tracing provider and :ref:`x-request-id<config_http_conn_man_headers_x-request-id>`
+    // never be modified.
```

Comment:
Not sure if I understand this correctly, but please let me know if I'm wrong here:
```suggestion
    // By default, tracing requires the :ref:`x-request-id <config_http_conn_man_headers_x-request-id>` header to be set, and modifies its value.
    // If true, the request will be traced, and *x-request-id* will not be modified.
```

Reply:
I think that we should add that `the request will bypass x-request-id based tracing decision`.

Reply:
What operation will be expected in the future?

Reply:
I was interested in your perspective on this.

Reply:
It can be changed as follows.

```
enum TraceRequestIdSampleDecision {
  None,
  Bypass, // It acts as `bypass_sampling_with_request_id` is true.
}
```

WDYT?

Reply:
Sure, with `NONE`, `BYPASS`. If you have high confidence we would never have some other trace sample strategy with request ID I'm fine with the current `bool`, just would like to make sure this is deliberative.

Reply:
done